src/Symfony/Component/DependencyInjection/Tests/Compiler/MergeExtensionConfigurationPassTest.php

```diff
@@ -194,3 +210,36 @@ public function load(array $configs, ContainerBuilder $container)
         throw new \Exception();
     }
 }
+
+final class TestCccConfiguration implements ConfigurationInterface
```

Comment:
not sure what the "ccc" is about :}

Reply:
The third placeholder value I use for tests after foo and bar 😬